applications/users/tests/test_user_registration.py

```diff
@@ -0,0 +1,82 @@
+from django.core import mail
+from django.test import TestCase
+import factory
+from rest_framework.test import APITestCase, APIClient, APIRequestFactory
+from rest_framework.views import status
+
+from .factories import UserFactory
+from applications.users.models import User
+
+class UserRegistrationTests(APITestCase):
+    url = '/api/v1/auth/registration/'
```

Comment:
I think you can use `reverse`  to avoid hardcoded strings (check https://www.django-rest-framework.org/api-guide/reverse/), WDYT?

Reply:
+1 to this. Check the use of `reverse` to avoid having to redefine the urls in every test file.